extra/libnetwork/drivers/bridge/setup_ipv4.go

```diff
@@ -16,13 +17,13 @@ func setupBridgeIPv4(config *networkConfiguration, i *bridgeInterface) error {
 		return fmt.Errorf("failed to retrieve bridge interface addresses: %v", err)
 	}
 
-	if !types.CompareIPNet(addrv4.IPNet, config.AddressIPv4) {
+	if os.Getenv("SetBridgeIP") == "true" && !types.CompareIPNet(addrv4.IPNet, config.AddressIPv4) {
```

Comment:
This change should add something into the document, right?
For this kind of work, the pouchd's source code relies on some outter specific environment variables. If no guidance is here, it is quite hard to software user to notice the usage of env. WDYT?
How to solve the problem for user is what we need to take into consideration. @rudyfly

Reply:
I will change it in my next job that vendor libnetwork into alibaba group.